fluent-bundle/src/types/mod.rs

```diff
@@ -253,6 +253,25 @@ impl<'source> FluentValue<'source> {
         }
     }
 
+    /// Converts the [`FluentValue`] to a string.
```

Comment:
Suggestion (docs): The docs here are identical to `as_string` and it took some detailed reading to understand the difference. Could you update the code comment here and above in `as_string` explaining the difference for the uses?

The key difference is that `into_string` takes ownership of `self`.

Reply:
The docs should be a little better now. Thanks for the reviews btw!